src/components/SharedComponents/SwitchComponent.style.tsx

```diff
@@ -7,24 +7,24 @@ export const SwitchLabel = styled(Grid)`
 `;
 
 export const SwitchGrid = styled(Grid)<{
-  viewAllCounties: boolean | undefined;
+  checked: boolean | undefined;
```

Comment:
I assume `viewAllCounties` was from before we made this a shared component.  Renamed to `checked` so it's more generic. 😁